Update docs for default 2 GB Kibana instances (#6121)
elastic/docs-content: deploy-manage/deploy/elastic-cloud/elastic-cloud-hosted-planning.md

To ensure optimal performance and cluster stability in your production environment, we recommend adhering to the following minimum size guidelines. Deviating from these recommendations may lead to performance issues and cluster instability. For an enhanced user experience, consider planning your deployment capacity above these minimum recommendations, and adjust sizing based on your specific use case.

* **{{es}} nodes/instances**: For production systems, we recommend that each {{es}} node/instance in your cluster has at least 4 GB of RAM.

> * **{{kib}} nodes/instances**: For production systems, we recommend that each {{kib}} node/instance in your cluster has [at least 2 GB of RAM](/deploy-manage/upgrade/deployment-or-cluster/upgrade-on-ech.md#check-your-kib-instance-size). {{kib}} instances in new {{ech}} deployments have 2 GB of RAM by default. 

Reviewer: we should not link to the upgrade guide here. we can possibly add the relevant details from the link target: 

> Increasing the memory of your instances ensures optimal performance of new and existing Platinum and Enterprise features. With less than 2 GB of RAM, you can face service interruptions when using features such as Security detection rules, reporting, workflows, and Agent Builder.

> * **{{kib}} nodes/instances**: For production systems, we recommend that each {{kib}} node/instance in your cluster has [at least 2 GB of RAM](/deploy-manage/upgrade/deployment-or-cluster/upgrade-on-ech.md#increase-your-kib-instance-size). {applies_to}`stack: ga 9.4+` {{kib}} nodes/instances in new {{ech}} deployments have 2 GB of RAM by default. 

Reviewer 2: ```suggestion
* **{{kib}} nodes/instances**: For production systems, we recommend that each {{kib}} node/instance in your cluster has [at least 2 GB of RAM](/deploy-manage/upgrade/deployment-or-cluster/upgrade-on-ech.md#increase-your-kib-instance-size). {{kib}} nodes/instances in new {{ech}} deployments have 2 GB of RAM by default. 
```
I think it's for all new deployments not just 9.4 from the moment the change is made to ECH

> * **{{kib}} nodes/instances**: For production systems, we recommend that each {{kib}} node/instance in your cluster has at least 2 GB of RAM. Below this size, Platinum and Enterprise features such as Security Detection Rules, Reporting, Workflows, and Agent Builder might be unavailable or cause service interruptions.  {{kib}} instances in new {{ech}} deployments have 2 GB of RAM by default.

Reviewer: ```suggestion
* **{{kib}} instances**: For production systems, we recommend that each {{kib}} instance in your cluster has at least 2 GB of RAM. Below this size, Platinum and Enterprise features such as Security Detection Rules, Reporting, Workflows, and Agent Builder might be unavailable or cause service interruptions.  {{kib}} instances in {{ech}} deployments created in May 2026 or later have 2 GB of RAM by default.
```

> * **{{kib}} instances**: For production systems, we recommend that each {{kib}} instance in your cluster has at least 2 GB of RAM. Below this size, Platinum and Enterprise features such as Security Detection Rules, Reporting, Workflows, and Agent Builder might be unavailable or cause service interruptions.  {{kib}} instances in {{ech}} deployments created in May 2026 or later have 2 GB of RAM by default.

Author: ```suggestion
* **{{kib}} instances**: For production systems, we recommend that each {{kib}} instance in your cluster has at least 2 GB of RAM. Below this size, Platinum and Enterprise features such as Security Detection Rules, Reporting, Workflows, and Agent Builder might cause service interruptions. {{kib}} instances in {{ech}} deployments created in May 2026 or later have 2 GB of RAM by default.
```
Updating based on Yuvi's comment [here](https://github.com/elastic/kibana/pull/264554#discussion_r3175136592)
